integration/examples/annotated-skaffold.yaml

```diff
@@ -77,7 +77,8 @@ build:
   # `useBuildkit` can also be set to activate the experimental BuildKit feature.
   #
   # local:
-  #   skipPush: true
+  #   false by default for local clusters, true for remote clusters
+  #   push: false
```

Comment:
The doc in the previous paragraph still talks about skip push, with inverted boolean value: this is confusing.